app/models/manageiq/providers/storage_manager/cinder_manager/refresh_parser/cross_linkers.rb

```diff
@@ -5,6 +5,10 @@ def self.cross_link(ems, data)
       _log.warn "Manager does not have a parent."
       return
     end
+    unless data
+      _log.warn "Manager does not have volumes, snapshots, or volume backups."
```

Comment:
Just duplicating the previous code.  Are you asking for me to go back and change existing code, or to have the new code be inconsistent with the old code?